docs/content/concepts.mdx

```diff
@@ -200,7 +200,7 @@ For more information, please see [Direct Access](./modeling/direct-access.mdx).
 
 ## What Is A Relation?
 
-A **relation** is a string defined in the type definition of an authorization model that defines the possibility of a relationship between objects of this type and other users in the system.
+A **relation** is a string defined in the type definition of an authorization model that defines the possibility of a relationship between objects and other users in the system.
```

Comment:
Sorry! Last time, but I think it would read nicer as:
```suggestion
A **relation** is a string defined in the type definition of an authorization model that defines the possibility of a relationship between an object and a user in the system.
```

Reply:
Please add an indicator that specifies that we are referencing objects of the same type as the type definition

Reply:
will write it as 
```
A **relation** is a string defined in the type definition of an authorization model that defines the possibility of a relationship between an object of the same type as the type definition and a user in the system.
```